deprecated_prepend_stack_test.go

```diff
@@ -0,0 +1,77 @@
+package microerror
+
+import "testing"
+
+func Test_DeprecatedPrependStack_1(t *testing.T) {
+	c := ErrorHandlerConfig{}
+	h := NewErrorHandler(c)
+
+	e1 := &Error{
+		Desc: "test desc 1",
+		Kind: "testKind1",
+	}
+	e2 := &maskedError{
+		Cause: withAnnotation(&Error{
+			Desc: "test desc 2",
+			Kind: "testKind2",
+		}, ""),
+		Stack: []maskedErrorStack{
+			{},
+			{},
+			{},
+		},
+	}
+
+	err := h.Maskf(e1, e2.Error())
+	err = DeprecatedPrependStack(err, e2)
```

Comment:
This is now `DeprecatedPrependStack`. Otherwise we'd lose the first stack trace set here: `err := h.Maskf(e1, e2.Error())`.

The alternative is to extend spec with something like `MaskStack(err *Error, cause error) error`.